exporters/jaeger/src/main/java/io/opentelemetry/exporters/jaeger/JaegerGrpcSpanExporter.java

```diff
@@ -39,11 +39,13 @@
   private static final Logger logger = Logger.getLogger(JaegerGrpcSpanExporter.class.getName());
   private static final String CLIENT_VERSION_KEY = "jaeger.version";
   private static final String CLIENT_VERSION_VALUE = "opentelemetry-java";
-  private static final String DEFAULT_JAEGER_ENDPOINT = "localhost:14250";
+  public static final String DEFAULT_ENDPOINT = "localhost:14250";
   private static final String HOSTNAME_KEY = "hostname";
-  private static final String UNKNOWN = "unknown";
+  private static final String DEFAULT_HOST_NAME = "unknown";
   private static final String IP_KEY = "ip";
   private static final String IP_DEFAULT = "0.0.0.0";
+  public static final String DEFAULT_SERVICE_NAME = DEFAULT_HOST_NAME;
```

Comment:
The obsessive in me would love to have the public constants next to each other ;)